var/spack/repos/builtin/packages/zfp/package.py

```diff
@@ -16,9 +16,8 @@ class Zfp(MakefilePackage):
     url      = 'http://computing.llnl.gov/projects/floating-point-compression/download/zfp-0.5.2.tar.gz'
 
     version('0.5.4', sha256='768a05ed9bf10e54ac306f90b81dd17b0e7b13782f01823d7da4394fd2da8adb')
-    version('0.5.2', '2f0a77aa34087219a6e10b8b7d031e77')
-    version('0.5.1', '0ed7059a9b480635e0dd33745e213d17')
-    version('0.5.0', '2ab29a852e65ad85aae38925c5003654')
+    version('0.5.2', sha256='89e718edb966422b5898b5c37b1b0a781d4effacb511520558469e3ff7f65d7c')
+    version('0.5.1', sha256='867c04cf965f1c70d9725b396c6e1b5d29db55b0d69b8e87a995aaebd221b830')
```

Comment:
I can confirm that this conversion from MD5 to SHA256 looks legit:
```console
$ md5 zfp-0.5.*
MD5 (zfp-0.5.1.tar.gz) = 0ed7059a9b480635e0dd33745e213d17
MD5 (zfp-0.5.2.tar.gz) = 2f0a77aa34087219a6e10b8b7d031e77
$ shasum -a 256 zfp-0.5.*
867c04cf965f1c70d9725b396c6e1b5d29db55b0d69b8e87a995aaebd221b830  zfp-0.5.1.tar.gz
89e718edb966422b5898b5c37b1b0a781d4effacb511520558469e3ff7f65d7c  zfp-0.5.2.tar.gz
```
Not that I don't trust you, I'm just not sure what our policy on checksum conversion is.